pkg/resources/resource_monitor.go

```diff
@@ -310,12 +345,65 @@ func UpdateResourceMonitor(d *schema.ResourceData, meta interface{}) error {
 		stmt.SetString(`END_TIMESTAMP`, d.Get("end_timestamp").(string))
 	}
 
+	// Set triggers
+	sTrigs := expandIntList(d.Get("suspend_triggers").(*schema.Set).List())
+	for _, t := range sTrigs {
+		runSetStatement = true
+		stmt.SuspendAt(t)
+	}
+	siTrigs := expandIntList(d.Get("suspend_immediate_triggers").(*schema.Set).List())
+	for _, t := range siTrigs {
+		runSetStatement = true
+		stmt.SuspendImmediatelyAt(t)
+	}
+	nTrigs := expandIntList(d.Get("notify_triggers").(*schema.Set).List())
+	for _, t := range nTrigs {
+		runSetStatement = true
+		stmt.NotifyAt(t)
+	}
+
 	if runSetStatement {
 		if err := snowflake.Exec(db, stmt.Statement()); err != nil {
 			return fmt.Errorf("error updating resource monitor %v\n%w", id, err)
 		}
 	}
 
+	// Remove from account
+	if d.HasChange("set_for_account") && !d.Get("set_for_account").(bool) {
```

Comment:
this could be combined with the add to account. something like

```
if d.HasChange("set_for_account")
  old, new := d.GetChange("set_for_account")
 if new == nil {
  // remove from account
} else {
 // set on new account
}
f d.Get("set_
```

Reply:
This was intentional the issue is you can't have it set for both an account and a warehouse. So if you are going from setting a resource monitor from the account level to the warehouse level or vice versa  you have to make sure it is unset first.

Reply:
in that case you should also use"ConflictsWith", it works like setting ForceNew on an attirbute. Will prevent people from setting both at the same time.

```
// ConflictsWith is a set of schema keys that conflict with this schema.
// This will only check that they're set in the _config_. This will not
// raise an error for a malfunctioning resource that sets a conflicting
// key.
ConflictsWith []string
```